src/schemas/json/unist.json

```diff
@@ -0,0 +1,158 @@
+          "minItems": 1
+        },
+        "data": {
+          "$ref": "#/definitions/dataDefinition"
+        },
+        "id": {
+          "$ref": "#/definitions/idDefinition"
+        },
+        "parent": {
+          "$ref": "#/definitions/branchDefinition"
+        },
+        "position": {
+          "$ref": "#/definitions/positionDefinition"
+        },
+        "type": {
+          "$ref": "#/definitions/typeDefinition"
+        },
+        "value": {
+          "not": {}
+        }
+      },
+      "required": ["children", "type"],
+      "additionalProperties": false
+    },
+    "childDefinition": {
+      "description": "A child is a direct descending nodes of another node.",
+      "type": "object",
+      "anyOf": [
+        { "$ref": "#/definitions/branchDefinition" },
+        { "$ref": "#/definitions/literalDefinition" }
+      ]
+    },
+    "dataDefinition": {
+      "description": "The data field represents information from the ecosystem. This space is guaranteed to never be specified by unist or specifications implementing unist.",
+      "type": "object"
+    },
+    "idDefinition": {
+      "description": "The node ID.",
+      "anyOf": [
+        {
+          "type": "integer",
+          "minimum": 1
+        },
+        {
+          "type": "string",
+          "minLength": 1
+        }
+      ]
+    },
+    "literalDefinition": {
+      "description": "A literal is a node with a value field.",
+      "type": "object",
+      "properties": {
+        "children": {
+          "not": {}
+        },
+        "data": {
+          "$ref": "#/definitions/dataDefinition"
+        },
+        "id": {
+          "$ref": "#/definitions/idDefinition"
+        },
+        "parent": {
+          "$ref": "#/definitions/branchDefinition"
+        },
+        "position": {
+          "$ref": "#/definitions/positionDefinition"
+        },
+        "type": {
+          "$ref": "#/definitions/typeDefinition"
+        },
+        "value": {
+          "description": "The value field can contain any value."
+        }
+      },
+      "required": ["type", "value"],
+      "additionalProperties": false
+    },
+    "pointDefinition": {
+      "description": "A point represents one place in a source file.",
+      "type": "object",
+      "properties": {
+        "column": {
+          "description": "The column field (1-indexed integer) represents a column in a source file.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "line": {
+          "description": "The line field (1-indexed integer) represents a line in a source file.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "offset": {
+          "description": "The offset field (0-indexed integer) represents a character in a source file.",
+          "type": "integer",
+          "minimum": 0
+        }
+      },
+      "required": ["column", "line"],
+      "additionalProperties": false
+    },
+    "positionDefinition": {
```

Comment:
I also personally prefer to avoid them in general within variable names (I was following tsconfig schema for this one too!), so I'll go for the PascalCase.